bin/wp-merge-to-cp.sh

```diff
@@ -0,0 +1,40 @@
+#/usr/bin/env bash
+
+# First parameter is your fork
+# Second parameter is the ticket merged
+
+if [ ! -d "ClassicPress" ]; then
+    # Download ClassicPress from your fork
+    git clone git@github.com:$1/ClassicPress.git
+    cd ClassicPress
+
+    # Download original ClassicPress
+    git remote add cp git@github.com:ClassicPress/ClassicPress.git
+
+    # Download WordPress dev
+    git remote add wp https://github.com/WordPress/wordpress-develop
+else
+    cd ClassicPress
+fi
+git fetch cp
+git fetch wp
+
+# Switch to ClassicPress branch
+git fetch origin
+git checkout develop
+
+# Sync your fork with the original
+git merge cp/develop
+git push origin develop
+git checkout origin/develop -B develop
+
+# Create branch with the patch from WordPress
+git checkout -b merge/wp-r$2
+
+# Get the commit from WP git log
+commit=`git log wp/master --grep=$2 --oneline --pretty=format:'%h' -n 1`
```

Comment:
`--grep=$2` will usually match the wrong commit.  This should be matching against the `https://develop.svn.wordpress.org/trunk@44297` line in the commit message instead.

`master` is not always the right branch to check.  A change could appear in any of the `4.9` `5.0`, `5.1`, or `master` branches.

We also probably don't want to check the full history of each branch.  It'll be better to check `git log ff6114f8...wp/master`.  Here you can find the correct "base" commit for each branch (still need to add `5.1` to this list):  https://github.com/ClassicPress/ClassicPress-Bots/blob/4c1a9f2f2aa753171abade7c45a459e1fb94c4b9/app/Http/Controllers/UpstreamCommitsList.php#L10-L36

Reply:
> still need to add `5.1` to this list

Never mind, the 5.1 branch hasn't been created yet so we are fine with these three options `4.9` `5.0` and `trunk`/`master`.

Reply:
I will do that changes